src/browsers/schema/content/about.md

```diff
@@ -0,0 +1,46 @@
+# About SCHEMA
+
+The Schizophrenia Exome Sequencing Meta-analysis (SCHEMA) consortium is a large multi-site collaboration dedicated to aggregating, generating, and analyzing high-throughput sequencing data of schizophrenia patients to improve our understanding of disease architecture and advance gene discovery. The consortium was formed in the summer of 2017 with a commitment of data sharing, diversity, and inclusivity - our hope is that the findings from this study and others like it will provide a foundation for further investigation of disease mechanism and therapeutic discovery. This browser is part of that overall effort to display and share these results with the wider scientific community.
+
+The SCHEMA Phase I dataset (GRCh37/hg19) contains the analysis of exomes from 24,248 cases and 97,322 controls, and _de novo_ mutations from 3,402 parent-proband trios. Our study has actively recruited from diverse global populations, and includes individuals of European, Latin American, East Asian, Ashkenazi Jewish, and African American ancestry. Because the sequence data was generated with various capture technologies over a span of seven years, we adapted and developed methods to reduce possible confounders, and incorporated this information during the quality control and analysis steps. The first results have provided genome-wide significant results associating ultra-rare protein-coding variants in individual genes to risk of schizophrenia. Later releases are planned with larger number of samples that will further increase power, and will be released here before any publication.
+
+TWe would also like to thank the many tens of thousands of patients and families who generously contributed to our effort. This project is made possible by the generosity of many funders, including the [Stanley Foundation](https://www.broadinstitute.org/stanley), [National Institutes of Mental Health (NIMH)](https://www.nimh.nih.gov/index.shtml), Kent and Elizabeth Dauten, and The Dalio Foundation, and the leadership of its members.
```

Comment:
Extra "T" at the start of this sentence.

Reply:
Fixed.